api/v1alpha1/clusterextension_types.go

```diff
@@ -168,11 +169,11 @@ type ClusterExtensionSpec struct {
 	//
 	// When unspecified, upgrade edges across all channels will be used to identify valid automatic upgrade paths.
 	//
-	// This field follows the DNS subdomain name standard as defined in RFC
-	// 1123, with a deviation in the maximum length being no more than 48
-	// characters. This means that valid values:
-	//   - Contain no more than 48 characters
-	//   - Contain only lowercase alphanumeric characters, '-', or '.'
+	// This field follows the DNS label name standard as defined in [RFC 1123]
+	// with the addition of '.' and '_' characters being allowed.
```

Comment:
Is this valid: `a._-b`?

If it is, I don't think it should be.

Reply:
I do believe that would be valid

Reply:
With the new use of the RFC 1123 DNS Subdomain pattern, something like `a.-b` is not valid. I will add another admission test for this.

Reply:
should be added now